src/PrestaShopBundle/Install/Install.php

```diff
@@ -293,18 +324,129 @@ public function clearDatabase($truncate = false)
+            $context->language = new Language('en');
+        }
+
+        if (!isset($context->country) || !\Validate::isLoadedObject($context->country)) {
+            if ($id_country = (int)Configuration::get('PS_COUNTRY_DEFAULT')) {
+                $context->country = new Country((int)$id_country);
+            }
+        }
+        if (!isset($context->currency) || !\Validate::isLoadedObject($context->currency)) {
+            if ($id_currency = (int)Configuration::get('PS_CURRENCY_DEFAULT')) {
+                $context->currency = new \Currency((int)$id_currency);
+            }
+        }
+
+        /* Instantiate cookie */
+        $cookie_lifetime = defined('_PS_ADMIN_DIR_') ? (int)Configuration::get('PS_COOKIE_LIFETIME_BO') : (int)Configuration::get('PS_COOKIE_LIFETIME_FO');
+        if ($cookie_lifetime > 0) {
+            $cookie_lifetime = time() + (max($cookie_lifetime, 1) * 3600);
+        }
+
+        $cookie = new \Cookie('ps-s'.$context->shop->id, '', $cookie_lifetime, 'localhost', false, false);
+
+        $context->cookie = $cookie;
+
+        $context->cart = new \Cart();
+        $context->employee = new Employee(1);
+        if (!defined('_PS_SMARTY_FAST_LOAD_')) {
+            define('_PS_SMARTY_FAST_LOAD_', true);
+        }
+        require_once _PS_ROOT_DIR_.'/config/smarty.config.inc.php';
+
+        $context->smarty = $smarty;
+    }
+
+    public static function createTestDB() {
+        define('_PS_IN_TEST_', true);
+        define('_PS_ROOT_DIR_', __DIR__ . '/../../..');
+        define('_PS_MODULE_DIR_', _PS_ROOT_DIR_.'/tests/resources/modules/');
+        require_once(__DIR__.'/../../../install-dev/init.php');
+
+        $install = new Install();
+        \DbPDOCore::createDatabase(_DB_SERVER_, _DB_USER_, _DB_PASSWD_, _DB_NAME_, false);
+        $install->clearDatabase();
+        $install->installDatabase();
+        $install->initializeTestContext();
+        $install->installDefaultData('test_shop', false, false, true);
+        $install->populateDatabase();
+        $install->installCldrDatas();
+
+        $install->configureShop(array(
+            'admin_firstname' => 'puff',
+            'admin_lastname' => 'daddy',
+            'admin_password' => 'test',
+            'admin_email' => 'test@prestashop.com',
+            'configuration_agrement' => true,
+            'send_informations' => false,
+        ));
+        $install->installFixtures();
+        $install->installTheme();
+        $language = new \Language(1);
+        Context::getContext()->language = $language;
+        $install->installModules();
+        $install->installModulesAddons();
+
+        DatabaseDump::create();
+    }
+
+    public static function restoreTestDB()
+    {
+        if (!file_exists(sys_get_temp_dir().'/'.'ps_dump.sql')) {
+            throw new DBALException('You need to run composer create-test-db to create the initial test database');
```

Comment:
Maybe use `"You need to run 'composer create-test-db' to create the initial test database"`?